external/fv3fit/fv3fit/_shared/config.py

```diff
@@ -81,7 +80,6 @@ class TrainingConfig:
     hyperparameters: Dataclass
     additional_variables: List[str] = dataclasses.field(default_factory=list)
     sample_dim_name: str = "sample"
-    random_seed: Union[float, int] = 0
```

Comment:
Why is this moved? The logic to set random seeds should be transferrable across all training codes, and is done in `fv3fit.train`. sklearn has an additional way of setting random seeds, but I think even if we didn't set it the top-level logic would make it deterministic.

Speaking of which, separate from this PR we could probably use a test that runs `fv3fit.train` twice and tests that the answer is the same.